cypress/e2e/awx/administration/management-jobs.cy.ts

```diff
@@ -123,3 +126,288 @@ describe('Management Jobs Page - List and Launch Jobs', () => {
     });
   });
 });
+
+describe('Management Jobs - Schedules Tab', () => {
+  beforeEach(() => {
+    cy.awxLogin();
+  });
+  const managementJobsList = [
+    'Cleanup Activity Stream',
+    'Cleanup Expired OAuth 2 Tokens',
+    'Cleanup Expired Sessions',
+    'Cleanup Job Details',
+  ];
+
+  const autoGeneratedSchedules = [
+    'Cleanup Activity Schedule',
+    'Cleanup Expired OAuth 2 Tokens',
+    'Cleanup Expired Sessions',
+    'Cleanup Job Schedule',
+  ];
+
+  managementJobsList.forEach((jobName, index) => {
+    it(`admin can see existing auto generated schedules of the management job: ${jobName}`, () => {
+      cy.intercept('GET', awxAPI`/system_job_templates/?order_by=name&page=1&page_size=10`).as(
+        'getManagementJobsListPage'
+      );
+      cy.navigateTo('awx', 'management-jobs');
+      cy.verifyPageTitle('Management Jobs');
+      cy.wait('@getManagementJobsListPage')
+        .its('response.body.results')
+        .then((results: SystemJobTemplate[]) => {
+          const jobId = results.find((job) => job.name === jobName)?.id;
+          if (jobId) {
+            cy.get(`[data-cy="name-column-cell"]`).contains(jobName).click();
+            cy.verifyPageTitle(jobName);
+            cy.clickTab('Schedules', true);
+            cy.get('[data-cy="name-column-cell"]')
+              .first()
+              .should('have.text', autoGeneratedSchedules[index])
+              .should('be.visible');
```

Comment:
Seeing failures when testing this out because the first column doesn't always seem to be the autogenerated schedule. Might want to use a more generic check such as `contains` that would check a specific component for the existence of that auto generated schedule name

Reply:
yep, noticing that locally. earlier when I ran them locally I didn't have so many resources and the tests passed. I'm working on the fix, thank you!